src/bci_build/package/python.py

```diff
@@ -45,14 +45,20 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
     py3_ver_replacement = f"%%py{py3_ver_nodots}_ver%%"
     pip3 = f"{py3}-pip"
     pip3_replacement = "%%pip_ver%%"
-    has_pipx = False
-    has_wheel = True if is_system_py or os_version == OsVersion.TUMBLEWEED else False
-    # Tumbleweed rocks
-    if os_version == OsVersion.TUMBLEWEED:
-        has_pipx = True
-    # Enabled only for Python 3.11 on SLE15 (jsc#PED-5573)
-    if os_version not in (OsVersion.BASALT, OsVersion.TUMBLEWEED) and py3_ver == "3.11":
+    has_pipx: bool = False
+    has_wheel: bool = is_system_py
+
+    if os_version.is_tumbleweed:
+        # Tumbleweed rocks
         has_pipx = True
+        has_wheel = True
+    elif os_version.is_sle15:
+        if py3_ver != "3.6":
+            # Enabled only for Python 3.11+ on SLE15 (jsc#PED-5573)
+            has_pipx = True
+        # py3.12 pending discussion
+        if py3_ver not in ("3.6", "3.12"):
+            has_wheel = True
```

Comment:
```suggestion
        # py3.12 pending discussion
        has_wheel = py3_ver == "3.11"
```

Reply:
yeah, I picked this style so that when we add a new version, it pulls in wheel, rather than not pulling in wheel (which happened now and was missed that way). hence I wanted it opt out, so that it fails on the next version addition rather than silently missing a feature.